reg_tests/test_files/airfoilRANSEdgeTrilinos.rst/airfoilRANSEdgeTrilinos.rst.i

```diff
@@ -68,17 +69,17 @@ realms:
 
         - WallDistance:
             name: myNDTW
-            max_iterations: 1
+            max_iterations: 2
```

Comment:
@overfelt Why were these iterations increased?

Reply:
I was trying things to get the CPU/GPU results to match better and just increased all iterations on everything.  Not sure it it helped for this particular calculation.